### PR TITLE
JAMES-3598 Summer dependency upgrade 2021

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: server/protocols/jmap-draft/pom.xml [hunk 1]
```diff
@@ -241,7 +241,7 @@
         <dependency>
             <groupId>org.jgrapht</groupId>
             <artifactId>jgrapht-core</artifactId>
-            <version>1.5.0</version>
+            <version>1.5.1</version>
```
Comment (Arsnael) [inline server/protocols/jmap-draft/pom.xml:244]: The commit is ` [UPGRADE] simpleclient_dropwizard 0.10.0 -> 0.11.0 `

This change is unrelated

**Review by Arsnael — COMMENTED** [review]
Don't forget to cleanup the revert and revert of revert commits that  just cancel each other and seem useless in the git history :)

File: server/protocols/jmap-draft/pom.xml (hunk 1 above)
Comment (chibenwa) [reply server/protocols/jmap-draft/pom.xml:244]: Mistake ;-)

**Comment by chibenwa** [comment]
:green_apple: build woo.

I did clean up the history.

**Review by Arsnael — APPROVED** [review]
